.github/workflows/package.yml

```diff
@@ -242,47 +242,53 @@ jobs:
           - label: Debian GNU/Linux bullseye
             id: debian-bullseye
             task-namespace: apt
-            target: debian-bullseye
-            test-docker-image: debian:bullseye
+            image: images:debian/11
           - label: Debian GNU/Linux bookworm
             id: debian-bookworm
             task-namespace: apt
-            target: debian-bookworm
-            test-docker-image: debian:bookworm
+            image: images:debian/12
           - label: Ubuntu Focal Fossa
             id: ubuntu-focal
             task-namespace: apt
-            target: ubuntu-focal
-            test-docker-image: ubuntu:focal
+            image: images:ubuntu/20.04
```

Comment:
We can remove this in a separated PR because Ubuntu 20.04 reached EOL.

Reply:
I have made a PR:

* #216